ext/storage_sources/gcp_store/gcp_connection.cpp

```diff
@@ -53,6 +55,14 @@ gcp_connection::put_object(const std::string &object_key, const std::string &fil
 int
 gcp_connection::delete_object(const std::string &object_key) const
 {
+    using namespace gcs;
+    gcs::Client client = _gcp_client;
+    google::cloud::Status status = client.DeleteObject(_bucket_name, object_key);
```

Comment:
we can use auto here.

Reply:
addressed here: 0adccbf3c97d00eb573380c9a26490d45ef417dc